packages/ra-core/src/controller/usePrevNextController.ts

```diff
@@ -107,8 +107,9 @@ export const usePrevNextController = <RecordType extends RaRecord = any>(
         linkType = 'edit',
         storeKey,
         limit = 1000,
-        sort = { field: 'id', order: SORT_ASC },
-        filter = {},
+        sort: initialSort = { field: 'id', order: SORT_ASC },
+        filter: permanentFilter = {},
+        filterDefaultValues = {},
```

Comment:
I don't undertand why we use two filters params as they seems to serve the same purpose. It's a bit confusing to use in my opinion